04_beerhouse/src/App.js

```diff
@@ -0,0 +1,12 @@
+import './App.css';
+import MainPage from './MainPage';
+
+function App() {
+  return (
+    <div className="App">
+      <MainPage></MainPage>
+    </div>
```

Comment:
```suggestion
    <MainPage />
```
self closing이 미래라고 하더라고

Reply:
`<div className="App">` 에서의 `className` 은 css 속성을 부여해주려 있는 값이라 서영이가 말해준 것처럼 지워줘도 좋아!